lib/benchee/formatters/html.ex

```diff
@@ -339,4 +250,79 @@ defmodule Benchee.Formatters.HTML do
       {:win32, _} -> "explorer"
     end
   end
+
+  # Major pages
+  EEx.function_from_file(:defp, :comparison, "priv/templates/comparison.html.eex", [
+    :input_name,
+    :suite,
+    :units,
+    :suite_json,
+    :inline_assets
+  ])
+
+  EEx.function_from_file(:defp, :job_detail, "priv/templates/job_detail.html.eex", [
+    :input_name,
+    :job_name,
+    :job_statistics,
+    :system,
+    :units,
+    :job_json,
+    :inline_assets
+  ])
+
+  EEx.function_from_file(:defp, :index, "priv/templates/index.html.eex", [
+    :names_to_paths,
+    :system,
+    :inline_assets
+  ])
+
+  # Partials
+  EEx.function_from_file(:defp, :head, "priv/templates/partials/head.html.eex", [:inline_assets])
+  EEx.function_from_file(:defp, :header, "priv/templates/partials/header.html.eex", [:input_name])
+
+  EEx.function_from_file(:defp, :js_includes, "priv/templates/partials/js_includes.html.eex", [
+    :inline_assets
+  ])
+
+  EEx.function_from_file(
+    :defp,
+    :version_note,
+    "priv/templates/partials/version_note.html.eex",
+    []
+  )
+
+  EEx.function_from_file(:defp, :input_label, "priv/templates/partials/input_label.html.eex", [
+    :input_name
+  ])
+
+  EEx.function_from_file(:defp, :data_table, "priv/templates/partials/data_table.html.eex", [
+    :statistics,
+    :units,
+    :options
+  ])
+
+  EEx.function_from_file(:defp, :system_info, "priv/templates/partials/system_info.html.eex", [
+    :system,
+    :options
+  ])
+
+  EEx.function_from_file(:defp, :footer, "priv/templates/partials/footer.html.eex", [
+    :dependencies
+  ])
```

Comment:
I like moving these down :+1: 

module is pretty long and we seem to draw a line between logic functions and rendering functions anyhow... might be a sign that this all could be moved to a separate module. However, it's a small plugin so not sure that's worth it :thinking: